### PR TITLE
Enable decoding of MP4/AAC/ALAC files

### PR DESCRIPTION
**Pull request by uklotzde**
This simple but effective workaround resolves the open issue.

I have only tested MP4/AAC so far.

### TODO

- [x] Rebase after #30 has been merged

### CODE REVIEW

File: decode_symphonia/src/lib.rs
Comment (Be-ing) [inline decode_symphonia/src/lib.rs:126]: unconditionally printing to stdout isn't great
```suggestion
                        log::debug!(
                            "Assuming {num_channels} channel(s) according to the first decoded packet",
                            num_channels = spec.channels.count()
                        );
```

Comment (uklotzde) [reply decode_symphonia/src/lib.rs:126]: I have seen println!() statements everywhere and simply adopted it. Of course, `log::debug` would be more suitable.

Comment (Be-ing) [reply decode_symphonia/src/lib.rs:126]: I do see a few more `println!` statements in the library. Changing those to also use `log::debug` would be nice.

Comment (uklotzde) [reply decode_symphonia/src/lib.rs:126]: ...but the `log` dependency is missing and first needs to be discussed. Should be introduced separately if desired and not by this PR.

Comment (uklotzde) [reply decode_symphonia/src/lib.rs:127]: [c644953](https://github.com/MeadowlarkDAW/creek/pull/22/commits/c64495311d5d6efab29bbd46da541fad16e76715)

Comment (Be-ing) [reply decode_symphonia/src/lib.rs:127]: #23

**Comment by BillyDM** [comment]
Oh yeah, did I give you merge privileges, or do I need to merge this myself? If the second one, is it ready to merge?

**Comment by uklotzde** [comment]
Ping

**Comment by Be-ing** [comment]
CI isn't working. Rebasing after merging #26 should fix it.

**Comment by Be-ing** [comment]
Try rebasing now

**Comment by uklotzde** [comment]
#28 will fix the CI errors.

**Comment by uklotzde** [comment]
Ready again.

**Review by orottier — APPROVED** [review]

**Review by Be-ing — APPROVED** [review]

**Comment by Be-ing** [comment]
@orottier Don't wait for my approval to merge unless there's something you specifically want my input on.

**Comment by orottier** [comment]
> @orottier Don't wait for my approval to merge unless there's something you specifically want my input on.

Alright!

**Pull request merged**
